git-annotate: add page (#5096)
tldr-pages/tldr: pages/common/git-annotate.md

# git annotate

> Show commit hash and last author on each line of a file.

> > Note: Use `git blame`.
> > This only difference between `git annotate` and `git blame` is that the former has a slightly different output format, and is used only for backwards-compatibility, and provide a more familiar command name for people coming from other SCM systems.

Reviewer: Perhaps this will be more concise?
```suggestion
> See `git blame`, which is preferred over `git annotate`.
> `git annotate` is provided for those familiar with other version control systems.
```

Author: So much better.

> More information: <https://git-scm.com/docs/git-annotate>.

> - Print file with author name and commit hash on each line:

Reviewer: Perhaps this will read better?
```suggestion
- Print a file with the author name and commit hash prepended to each line:
```

> `git annotate {{file}}`

Reviewer: Perhaps this would be more consistent with other existing pages?
```suggestion
`git annotate {{path/to/file}}`
```

> - Print file with author email and commit hash on each line:

Reviewer: Perhaps this will read better?
```suggestion
- Print a file with the author email and commit hash prepended to each line:
```

> `git annotate -e {{file}}`

Reviewer: Same here.
```suggestion
`git annotate -e {{path/to/file}}`
```
